src/libcore/iter.rs

```diff
@@ -2553,10 +2553,7 @@ pub trait Extend<A> {
     ///
     /// # Examples
     ///
-    /// Basic usage:
```

Comment:
This is becoming more common throughout the standard library as I go through things, so it should stay.